precompile/contract/contract.go

```diff
@@ -77,7 +98,12 @@ func (s *statefulPrecompileWithFunctionSelectors) Run(accessibleState Accessible
 	functionInput := input[SelectorLen:]
 	function, ok := s.functions[string(selector)]
 	if !ok {
-		return nil, suppliedGas, fmt.Errorf("invalid function selector %#x", selector)
+		return nil, suppliedGas, InvalidFunctionErr(selector)
```

Comment:
can we get rid of this function and just return the formatted error created by the function?